packages/core/src/builder.class.ts

```diff
@@ -2077,10 +2077,28 @@ export class Builder {
     return this.queueGetContent(modelName, options);
   }
 
-  getAll(modelName: string, options: GetContentOptions = {}): Promise<BuilderContent[]> {
-    return this.getContent(modelName, {
-      limit: 30,
-      ...options,
-    }).promise();
+  getAll(
+    modelName: string,
+    options: GetContentOptions & {
+      req?: IncomingMessage;
+      res?: ServerResponse;
+      apiKey?: string;
+    } = {}
+  ): Promise<BuilderContent[]> {
+    let instance: Builder = this;
+    if (!Builder.isBrowser && (options.req || options.res)) {
+      instance = new Builder(options.apiKey || this.apiKey, options.req, options.res);
+    } else {
+      if (options.apiKey && !this.apiKey) {
+        this.apiKey = options.apiKey;
```

Comment:
Shouldn't be
```suggestion
        instance.apiKey = options.apiKey;
```

Reply:
when the above condition isn't met (`!Builder.isBrowser ... `) `instance === this`